src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.6.json

```diff
@@ -98,11 +98,61 @@
 			"DisplayType": "PagesWithSections",
 			"ActivatedByPageId": null
 		},
+		{
+			"PageId": "7610A",
+			"SequenceId": null,
+			"SectionId": null,
+			"Title": "2. Delivering training in sectors",
+			"LinkTitle": "Agriculture, environmental and animal care",
+			"InfoText": "",
+			"Questions": [{
+					"QuestionId": "DAT-7610A-1",
+					"QuestionTag": "DATAgricultureHowManyStarts",
+					"Label": "How many starts does your organisation forecast in the 'Agriculture, environmental and animal care' sector in the first 12 months of joining the RoATP?",
+					"ShortLabel": "",
+					"QuestionBodyText": "",
+					"Hint": "Enter number of starts",
+					"Input": {
+						"Type": "Text",
+						"InputClasses": "govuk-input--width-4",
+						"Validations": [{
+								"Name": "Required",
+								"Value": "",
+								"ErrorMessage": "Enter number of starts in this sector"
+							},
+							  {
+								"Name": "Regex",
+								"Value": "^[0]*[1-9]{1}[0-9]{0,2}$",
+								"ErrorMessage": "Enter number of starts in this sector using 3 digits or less"
+							  }
+						]
+					}
+				}
+			],
+			"PageOfAnswers": [],
```

Comment:
Please remove the duplicate PageOfAnswers

Reply:
that's weird, I linted this and that didn't come up.... I must have hit the duplicate line button just before commit

Reply:
fixed  now